dev-python/cryptography/cryptography-3.3.2.ebuild

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Now I'm confused. This ebuild shouldn't be needed by anything. In fact, I think I can remove the old version already.

Reply:
Sorry, did not actually look into reverse-dependencies. If we don't need it anymore, that's good.